### PR TITLE
Bootstrap member edit area

### PR DESCRIPTION
**Pull request by KimberleyCook**
Added the required option to the fields that are required

screencapture-localhost-3000-member-edit-2020-12-09-18_38_01

### CODE REVIEW

File: app/views/members/_new.html.haml [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-    .large-6.columns
+    .col-sm-12.col-md-6
       = f.input :pronouns, label: "What pronouns do you use?", placeholder: "e.g. she/her/they"
 
   .row
-    .large-6.columns
-      = f.input :email
-    .large-6.columns
+    .col-sm
+      = f.input :email, required: true
```
Comment (despo) [inline app/views/members/_new.html.haml:13]: I wonder why these were not picked up automatically by simple_form 🍤

**Review by despo — APPROVED** [review]
👍🏼

**Comment by despo** [comment]
@KimberleyCook you should rebase your base branch to pick up the new CI and make sure you don't divert too much.

**Comment by KimberleyCook** [comment]
@despo rebased the staging bootstrap branch and this one 💃

**Pull request merged**